### PR TITLE
Add new validators

### PR DESCRIPTION
**Pull request by agraham19**
* Add new validators to the controller

### CODE REVIEW

**Comment by agraham19** [comment]
This commit adds min, max, between, ukPhoneNumber, decimal, and over18 validators.

File: lib/validation/validators.js [hunk 1]
```diff
@@ -65,6 +82,11 @@ module.exports = Validators = {
     return value === '' || Validators.regex(value, /^\(?\+?[\d()-]{0,15}$/);
   },
 
+  ukPhoneNumber(value) {
```
Comment (robertdeniszczyc2) [inline lib/validation/validators.js:85]: Should this be gbPhoneNumber as UK !== GB? Not sure if Northern Ireland phone numbers follow a different format to GB.

File: test/spec/spec.validators.js [hunk 2]
```diff
@@ -625,6 +625,148 @@ describe('Validators', () => {
...
+        });
+      });
+    });
+  });
+
+  describe('over18', () => {
+    describe('invalid values', () => {
+      const inputs = [
+        '2014-11-05',
+        '1996-11-06'
```
Comment (robertdeniszczyc2) [inline test/spec/spec.validators.js:749]: What about 1996-11-05? On your 18th birthday would you be classed as over 18?

File: lib/validation/validators.js (hunk 1 above)
Comment (agraham19) [reply lib/validation/validators.js:85]: Confusingly, the two-letter country code for the whole UK is GB, except for in domain names. NI also falls under +44.

File: test/spec/spec.validators.js (hunk 2 above)
Comment (agraham19) [reply test/spec/spec.validators.js:749]: I think as we're only working with dates and not times, we'd have to assume you become 18 at midnight. We have a check for the 18th birthday in the valid inputs section.

Comment (robertdeniszczyc2) [reply test/spec/spec.validators.js:749]: Apologies I read the test the wrong way round, yes this makes sense

**Review by robertdeniszczyc2 — APPROVED** [review]

**Pull request merged**